mbed_app.json

```diff
@@ -317,6 +317,34 @@
             "esp8266.cts"                               : "D12",
             "esp8266.rts"                               : "PB_1",
             "esp8266.rst"                               : "D8"
+        },
+        "WIO_3G": {
+            "target.features_add"                       : ["LWIP", "STORAGE", "BOOTLOADER"],
+            "target.macros_add"                         : ["MBEDTLS_USER_CONFIG_FILE=\"mbedTLSConfig_mbedOS.h\""],
+            "target.device_has_remove"                  : ["SERIAL_FC"],
+            "target.network-default-interface-type"     : "CELLULAR",
+            "target.components_add"                     : ["FLASHIAP"],
+            "target.bootloader_img"                     : "bootloader/mbed-bootloader-wio_3g-internal_flash-no_rot-v4.1.2.bin",
+            "target.header_offset"                      : "0x8000",
+            "target.app_offset"                         : "0x8400",
```

Comment:
target.restrict_size should be added to prevent overflowing to second bank.

Reply:
Thanks.  I have added target.restrict_size for this.